vmm/sandbox/build.rs

```diff
@@ -0,0 +1,5 @@
+extern crate built;
```

Comment:
1. Should add our Kuasar Author file header.
2. What is the usage of `extern crate built`?